content/applications/inventory_and_mrp/inventory/routes/strategies/putaway.rst

```diff
@@ -1,54 +1,209 @@
+correct location in the :guilabel:`Inventory Report`. View all products in stock in
+:menuselection:`Inventory --> Reporting --> Inventory Report`. By default, the products are
+grouped by location.
+
+Select the product, and the drop-down displays the locations the products currently are. In the
+image below, the locations shown are `WH/Stock/Fruits` and `WH/Stock/Vegetables`, which
+confirms that the putaway rules successfully.
+
+.. image:: putaway/destination-locations.png
    :align: center
+   :alt: Verify stock moves triggered by putaway rule
+
+For a more detailed report, click on a stock move line. On the report, the :guilabel:`Reference`
+displays the warehouse operation associated to the :guilabel:`Product`. These can be any operation
+involving movement of stock, such as manufacturing orders, inventory adjustments, or internal
+transfers. The :guilabel:`Source` links the document associated to the reference operation. Use the
+:guilabel:`From` and :guilabel:`To` location and the associated documents to trace the path of the
+product.
 
-.. image:: putaway/putaw5.png
+.. _inventory/putaway/wrong-location:
+
+Trace products in wrong locations
+---------------------------------
+
+Pictured below is a detailed stock move of the :guilabel:`Product`, `Apple` that was put away in the
+wrong location. This report is useful to trace the path the product moved and identify the cause of
+the wrong stock move.
+
+Start by selecting the stock move record for `Apple`. Here, a putaway rule was purposely set up
+incorrectly, so the product was moved from `WH/Stock` to `WH/Stock/Shelf 1` instead of the expected
+location, `WH/Stock/Fruits`.
+
+.. image:: putaway/stock-move-report.png
    :align: center
+   :alt: Open stock move report
+
+The :guilabel:`To` field displays where the product currently is, `WH/Stock/Shelf 1`, which is the
+wrong location. The :guilabel:`From` location is the vendor's location, which is expected since the
+:guilabel:`Source` of the stock move is a :abbr:`PO (Purchase Order)`.
+
+Diagnose why the apple ended up in the shelf by following the :guilabel:`Reference` warehouse
+operation in :menuselection:`Inventory --> Operations --> Transfers`. This is the page that displays
+all warehouse transfers-- internal or external. Click on the desired record.
+
+.. image:: putaway/transfers.png
+   :align: center
+   :alt: Select the desired record
+
+.. tip::
+   When there are too many records, use either the :guilabel:`Filters` or :guilabel:`Group by`
+   features to sort and :ref:`find desired records<inventory/putaway/use-advanced-filters>`.
+
+On the warehouse reception document, the field :guilabel:`Destination Location` shows where the
+product is received upon entering the warehouse.
+
+Clicking on the :guilabel:`Detailed Operations` icon (4 horizontal lines) to the right of the
+product opens a window that shows the product was moved to the wrong location. Based on this
```

Comment:
You might want to modify how this icon is referred to, per our writers workshop meeting the other day.

Also changing it to "product line" for clarity.
```suggestion
Clicking on the :guilabel:`Detailed Operations` icon (4 horizontal lines) to the right of the
product line opens a window that shows the product was moved to the wrong location. Based on this
```